src/app/dashboard/dashboard.component.ts

```diff
@@ -37,15 +37,23 @@ export class DashboardComponent implements OnInit {
   ngOnInit() {
     this.connectionStore.loadAll();
     this.integrationStore.loadAll();
+
     /**
      * If guided tour state is set to be shown (i.e. true), then show it for this page, otherwise don't.
      */
     if (this.userService.getTourState() === true) {
       this.tourService.initialize([ {
-          anchorId: 'dashboard.navigation',
+        route: 'dashboard',
```

Comment:
tabs seem off here

Reply:
Once merged, the new linter will take care of raising an alert for this upon committing code :)